tests/integration/debug_files/create_jvm_based_bundle.rs

```diff
@@ -0,0 +1,83 @@
+use crate::integration::{
+    copy_recursively, mock_common_upload_endpoints, register_test, ServerBehavior,
+};
+use std::fs::{create_dir, remove_dir_all, write};
+
+#[test]
+fn command_create_jvm_based_bundle_help() {
+    #[cfg(not(windows))]
+    register_test("debug_files/debug_files-create-jvm-based-bundle-help.trycmd");
+    #[cfg(windows)]
+    register_test("debug_files/debug_files-create-jvm-based-bundle-help-windows.trycmd");
```

Comment:
You don't need separate test cases for windows and non-windows here. The `windows` test case should work for both.

Reply:
Updated, let's see if CI is happy :-)